chirp/drivers/tk760.py

```diff
@@ -739,26 +736,26 @@ def get_settings(self):
         basic.append(clone)
 
         # front keys
-        rs = RadioSettingValueList(KEYS.values(),
-                                   KEYS.values()[KEYS.keys().index(
+        rs = RadioSettingValueList(list(KEYS.values()),
```

Comment:
We might should make these actually `list()`ify their parameters so we don't have to do this everywhere.

Reply:
I will need some help with this!

Reply:
I don't usually say this but, I think if we let these slide through, we should take the opportunity to come back around and simplify the logic, it's really going in circles here.

Take for example in the [tk760g.py driver](https://github.com/sv1/chirp/blob/8a7cf8e6a0b5a5ce0a7ab4bab5002216b2e290f4/chirp/drivers/tk760g.py#L1307) where I added in some print statements to see how full circle we actually go:

```python3
            # Here's what happens
            print KEYS
            print keys.kVOL_UP
            print int(keys.kVOL_UP)
            print KEYS.keys().index(int(keys.kVOL_UP))
            print KEYS.values()[KEYS.keys().index(int(keys.kVOL_UP))]
            print type(keys.kVOL_UP)
            # Here's what we could do instead
            print KEYS[int(keys.kVOL_UP)]
            
            vu = RadioSetting("keys.kVOL_UP", "VOL UP",
                              RadioSettingValueList(KEYS.values(),
                              KEYS.values()[KEYS.keys().index(
                                  int(keys.kVOL_UP))]))
```

```console
{161: 'Channel Up/Down', 51: 'Display character', 53: 'Home Channel', 55: 'CH down', 56: 'CH up', 57: 'Key lock', 58: 'Lamp', 59: 'Public address', 60: 'Reverse', 61: 'Horn alert', 62: 'Selectable QT', 63: '2-tone encode', 64: 'Monitor A: open mommentary', 65: 'Monitor B: Open Toggle', 66: 'Monitor C: Carrier mommentary', 67: 'Monitor D: Carrier toggle', 68: 'Operator selectable tone', 69: 'Redial', 70: 'RF Power Low', 71: 'Scan', 72: 'Scan del/add', 74: 'GROUP down', 75: 'GROUP up', 79: 'None', 80: 'VOL down', 81: 'VOL up', 82: 'Talk around', 93: 'AUX'}
0x51
81
25
VOL up
<type 'instance'>
VOL up
```

That line and many, many others would simply to just
```
            vu = RadioSetting("keys.kVOL_UP", "VOL UP",
                              RadioSettingValueList(KEYS.values(),
                              KEYS[int(keys.kVOL_UP)])
```

I'm more than happy to help but wanted to mention this before we "fix" `./chirp/settings.py`

Tony

Reply:
I'm working on it now and I think I'm addressing this concern. I'll add you to the PR so you can look.